core/src/test/scala/akka/persistence/cassandra/EventsByTagMigrationSpec.scala

```diff
@@ -377,13 +378,20 @@ abstract class AbstractEventsByTagMigrationSpec
     PersistenceQuery(systemThree).readJournalFor[CassandraReadJournal](CassandraReadJournal.Identifier)
 
   override protected def beforeAll(): Unit = {
-    super.beforeAll()
-    system.log.debug("Creating old tables, first dropping {}", messagesTableName)
-    // Drop the messages table as we want to start with the old one
-    cluster.execute(s"drop table $messagesTableName")
-    cluster.execute(oldMessagesTable)
-    cluster.execute(oldMateterializedView)
-    system.log.debug("Old tables created")
+    if (!CassandraSessionRegistry(system)
+          .sessionFor("alpakka.cassandra", system.dispatcher)
```

Comment:
why not use "our" plugin defaults "akka.persistence.cassandra"?

Reply:
👍